src/components/dialogs/network-modifications/generator/modification/generator-modification-dialog.js

```diff
@@ -201,6 +211,7 @@ const GeneratorModificationDialog = ({
                     equipmentId: editData?.regulatingTerminalId?.value,
                     equipmentType: editData?.regulatingTerminalType?.value,
                     voltageLevelId: editData?.regulatingTerminalVlId?.value,
+                    ...getPropertiesFromModification(editData.properties),
                 }),
```

Comment:
You should put this below, outside of parentheses, it's not taken into account here

Reply:
i fixed it